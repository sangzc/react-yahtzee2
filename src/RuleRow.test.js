import React from 'react';
import { mount } from "enzyme";
import RuleRow from "./RuleRow"
import toJson from "enzyme-to-json"

it('renders without crashing', () => {
    mount(<RuleRow />);
});

it('matches snapshot', () => {
    let wrapper = mount(<RuleRow />);
    let serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
});

