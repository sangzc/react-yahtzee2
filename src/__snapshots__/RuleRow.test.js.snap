// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<RuleRow>
  <tr
    className="RuleRow RuleRow-active"
  >
    <td
      className="RuleRow-name"
    />
    <td
      className="RuleRow-score"
    />
  </tr>
</RuleRow>
`;
